algoliasearch-core/src/main/java/com/algolia/search/AnalyticsConfig.java

```diff
@@ -16,7 +17,7 @@
      * @param apiKey The API Key
      */
     public Builder(@Nonnull String applicationID, @Nonnull String apiKey) {
-      super(applicationID, apiKey, createDefaultHosts());
+      super(applicationID, apiKey, createDefaultHosts(), CompressionType.NONE);
```

Comment:
nit: why is this part of the subclass constructor signature?

Reply:
As discussed with @Ant-hem: I missed the point where the `CompressionType.NONE` could actually not be used as default. Sorry for the noise.